# HG changeset patch
# User Yury G. Kudryashov <urkud@ya.ru>
# Date 1316994786 -14400
# Node ID 4c3cd9a4b0602c45c11ec62bcc1f5562ea071d70
# Parent  bb611e7cb1e58cc37a6773a22a15c74b55e3d267
Fix compilation on Linux

Submitted upstream, see http://bugzilla.libsdl.org/show_bug.cgi?id=1179

diff -r bb611e7cb1e5 -r 4c3cd9a4b060 IMG_jpg.c
--- a/IMG_jpg.c	Thu Aug 25 02:24:35 2011 -0400
+++ b/IMG_jpg.c	Mon Sep 26 03:53:06 2011 +0400
@@ -34,7 +34,7 @@
 
 #include <jpeglib.h>
 
-#if JPEG_LIB_VERSION >= 80
+#ifdef JPEG_TRUE /* MinGW version of jpeg-8.x renamed TRUE to JPEG_TRUE etc. */
 	typedef JPEG_boolean boolean;
 	#define TRUE JPEG_TRUE
 	#define FALSE JPEG_FALSE
